### PR TITLE
Official seed

### PR DESCRIPTION
**Pull request by amhursh**
Related Issue: #98 #134 

Summary: Fix Seed. Update Welcome carousel

Author(s): amhursh

### CODE REVIEW

File: app/views/welcome/index.html.erb [hunk 1]
```diff
@@ -13,26 +28,26 @@
...
 
       <div class="item">
-        <img src="https://baconmockup.com/900/300/" alt="New York" style="width:100%;">
+        <%= image_tag("headphones.jpg") %>
         <div class="carousel-caption">
-          <h3>New York</h3>
-          <p>We love the Big Apple! But we hat actual apples. Thanks Monsanto!</p>
+          <h1>Headphones</h1>
+          <h3>We aint got em!</h3>
         </div>
```
Comment (randy-darsh) [inline app/views/welcome/index.html.erb:51]: good use of actual images. although i gotta say. i was partial to the steak.

File: app/views/items/show.html.erb [hunk 2]
```diff
@@ -10,7 +10,6 @@
     <h1 class="h-tag-no-top-margin-title"> <%= @item.title %></h1>
     <h4>Price: $<%= @item.formatted_price %> each</h4>
     <p><%= @item.description %></p>
-    <p>DUMMY DESCRIPTION TEXT FOR FILLER. Spicy jalapeno bacon ipsum dolor amet ad turkey ea tail filet mignon occaecat ham hock pig aute tri-tip cupim meatloaf exercitation beef. Ground round tempor kevin cupidatat pork dolor turkey picanha cupim dolore ham hock shankle reprehenderit. Ut ground round bacon doner adipisicing kevin andouille shankle beef dolor. Ground round dolore cow do pork belly beef ribs sausage pastrami mollit adipisicing veniam ex enim rump dolore. Rump do mollit prosciutto jerky. Nostrud ut veniam pancetta bresaola short loin ball tip shoulder. Et cupidatat pork salami tail cow. Eu turkey nostrud shank bresaola. Tri-tip reprehenderit incididunt enim.</p>
```
Comment (randy-darsh) [inline app/views/items/show.html.erb:13]: good catch

File: app/assets/stylesheets/application.css [hunk 3]
```diff
@@ -34,7 +34,9 @@ body > header > nav > div > div.navbar-header > a > img {
   margin-top: -10px;
 }
 .item-card-img {
-  width: 95%;
+  width: 250px;
+  height: 250px;
+  margin-left: 20px;
   border-radius: 5px;
```
Comment (randy-darsh) [inline app/assets/stylesheets/application.css:40]: fixed the items index. good work

File: db/seeds.rb [hunk 4]
```diff
@@ -1,26 +1,113 @@
...
+													price: 299.60, category_id: dj.id, image: File.open(Rails.root.join('app/assets/images/item_images', 'gemini.png')))
+OrdersItem.where(item_id: 23).update(unit_price: 299.60)
+aaron_order2.items.create(title: "Numark", description: "The Numark NV DJ controller and Dashboard display bundle adds a visual element to your DJ mixes. NV reimagines the hardware-software experience for the professional DJ. And what better way to enhance the performance of the NV then with the real-time visual feedback of Serato DJ via Dashboards three vibrant, 4.3 full-color screens. Get them both with just one purchase and be on your way to mastering digital mixes.",
+													price: 998.0, category_id: dj.id, image: File.open(Rails.root.join('app/assets/images/item_images', 'numark.png')))
+OrdersItem.where(item_id: 24).update(unit_price: 998.0)
 
-  puts "Created #{item.title} with category #{item.category.title}"
-end
+puts "Created #{Item.count} of 24 items with #{OrdersItem.count} join rows"
```
Comment (randy-darsh) [inline db/seeds.rb:113]: this was essential. good work aaron

File: app/views/welcome/index.html.erb [hunk 5]
```diff
@@ -1,6 +1,21 @@
...
+<html>
+<head>
+  <title></title>
 
-<div class="container">
+  <link href="https://fonts.googleapis.com/css?family=Droid+Sans+Mono|Shrikhand" rel="stylesheet">
+
+  <style>
+  h1 {
+    font-family: 'Shrikhand', cursive;
```
Comment (limsammy) [inline app/views/welcome/index.html.erb:10]: Interested to see what this font looks like :3

**Review by limsammy — APPROVED** [review]
Gonna mergeatronic

**Pull request merged**